models/classes/class.QtiTestService.php

```diff
@@ -1142,12 +1144,13 @@ public function createContent(core_kernel_classes_Resource $test, $createTestFil
 
     private function createTestIdentifier(string $testLabel): string
     {
-        $identifier = Format::sanitizeIdentifier($testLabel);
-        $identifier = str_replace('_', '-', $identifier);
-        if (preg_match('/^\d/', $identifier)) {
```

Comment:
shouldn't we remove this condition instead?

Reply:
This condition was ensuring the first digit will be prefixed by '-'. We woould like to modify that by adding letter 't' at the begining of a identifier so it will start from letter always.